TYPO3.Neos/Resources/Public/JavaScript/Content/Navigate/AbstractNodeTree.js

```diff
@@ -156,7 +156,7 @@ define(
 
 			NewPositionSelectorButton: AbstractPositionSelectorButton.extend({
 				allowedPositionsBinding: 'parentView.allowedNewPositions',
-				title: 'Create (hold to select position)',
+				title: I18n.translate('TYPO3.Neos:Main:content.navigate.createNewHoldPosition', 'Create new (hold to select position)'),
```

Comment:
I mean stuff like this, it still appeared in english even though I switched to German, but maybe I did something wrong?